src/components/styles/StyledStartButton.js

```diff
@@ -1,8 +1,6 @@
 import styled from "styled-components";
 
-// because of the keyboard functionality of space (accept ) and enter (submit) to a button, we used a div in these place to prevent these.
-
-export const StyledStartButton = styled.div`
+export const StyledStartButton = styled.button`
```

Comment:
I'll close this PR, ok? we have the button set to .div on the main

Reply:
ok